src/python/examples/streamlined_example.py

```diff
@@ -0,0 +1,24 @@
+import tritonserver
+from tritonfrontend import KServeGrpc, KServeHttp
+
+model_path = "/root/models"
+server_options = tritonserver.Options(
+    server_id="ExampleServer",
+    model_repository=model_path,
+    log_error=True,
+    log_warn=True,
+    log_info=True,
+)
+server = tritonserver.Server(server_options).start()
+
+http_options = KServeHttp.Options()
+http_service = KServeHttp.Server(server, http_options)
+http_service.start()
+
+grpc_options = KServeGrpc.Options()
+grpc_service = KServeGrpc.Server(server, grpc_options)
+grpc_service.start()
+
+
+http_service.stop()
+grpc_service.stop()
```

Comment:
Include an explicit `server.stop()` to avoid known exit timeout issue

Reply:
do we want to have a context manager for services ?

Reply:
Latest changes should include an __enter__ and __exit__ function for each `FrontendService.Server` class. With this new flow a service can be started and stopped through:

```
server = ...
with KServeHttp.Server(server) as http_service:
       ...
server.stop()
```